devicetypes/erocm123/switch-child-device.src/switch-child-device.groovy

```diff
@@ -0,0 +1,49 @@
+/**
+ *  Switch Child Device
+ *
+ *  Copyright 2017 Eric Maycock
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Switch Child Device", namespace: "erocm123", author: "Eric Maycock") {
```

Comment:
Could we maybe move this to the smartthings namespace to make it easier to reuse for other similar devices?